s3fs/core.py

```diff
@@ -545,6 +545,35 @@ def seek(self, loc, whence=0):
         self.loc = nloc
         return self.loc
 
+    def readline(self, length=-1):
+        '''
+        Read and return a line from the stream.
+        
+        If size is specified, at most size bytes will be read.
```

Comment:
Maybe want to change `size` to `length`. I copied that from the `io` docs, which use `size` for the positional argument name, but s3fs uses `length`.